.travis.yml

```diff
@@ -6,5 +6,6 @@ env:
     - OCAML_VERSION=4.02 PACKAGE=key-parsers
     - OCAML_VERSION=4.03 PACKAGE=key-parsers
     - OCAML_VERSION=4.04 PACKAGE=key-parsers
+    - REVDEPS="pkcs11"
```

Comment:
That's supposed to read `OCAML_VERSION_4.04 PACKAGE=key-parsers REVDEPS="pkcs11"`.